### PR TITLE
Apply changes for the upcoming build infrastructure improvements (2.x)

### PR DESCRIPTION
**Pull request by warpech**
point 8 assumes changes from: https://github.com/Starcounter/AdminTrack/issues/438

point 4 assumes changes from: https://github.com/Starcounter/AdminTrack/issues/435

@Starcounter/webplatformteam, @ubbeK, @joozek78 please review.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-This repo is supposed to replace [/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles](https://github.com/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles). At this point it does not replace it yet, but it is the source of truth for it.
+This repo is the source of client files for [/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles](https://github.com/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles).
```
Comment (ubbeK) [inline CONTRIBUTING.md]: The link is wrong, this should not exist anymore (I can see that not all ClientFiles are removed though, will create an issue for that: https://github.com/Starcounter/level1/tree/develop/src/BuildSystem/ClientFiles)

I think you should say something like, source of client files found in `%StarcounterBin%/ClientFiles`

Comment (ubbeK) [reply CONTRIBUTING.md]: Issue has been created to remove remaining ClientFiles from Level1: https://github.com/Starcounter/level1/issues/4485

Comment (warpech) [reply CONTRIBUTING.md]: Thanks, fixed.

**Review by tomalec — APPROVED** [review]
LGTM

**Comment by warpech** [comment]
@ubbeK I made a few clarifications, especially changed the last point to be:

> 8. Merge the PR to StarcounterClientFiles branch `2.x`.
>   - at this point, a new build of Level1 is started

Is this correct? Does Level1 have a trigger on that?

**Review by ubbeK — APPROVED** [review]

**Pull request merged**